### ISSUE TITLE
Questie crash when killing Chill Nymph

### ISSUE THREAD

**Issue by Gymson64** [Questie/Questie#4831]
## Bug description

After completing the quest The Fallen Sisters (11314), every time I kill a Chill Nymph questie crashes and requires a reload. Bug description below:

> [ADDON_ACTION_BLOCKED] AddOn 'Questie' tried to call the protected function 'Questie_ItemButton4:SetAttribute()'.
> [string "@!BugGrabber/BugGrabber.lua"]:480: in function <!BugGrabber/BugGrabber.lua:480>
> [string "=[C]"]: in function `SetAttribute'
> [string "@Questie/Modules/Tracker/TrackerLinePool.lua"]:450: in function `SetItem'
> [string "@Questie/Modules/Tracker/QuestieTracker.lua"]:613: in function `Update'
> [string "@Questie/Modules/Quest/QuestEventHandler.lua"]:303: in function <...eQuestie/Modules/Quest/QuestEventHandler.lua:302>

## Questie & WoW version

Questie v8.0.1
WOTLK Classic

**Comment by MyTechnoHunter** [comment]
pretty sure this is fixed now just waiting on hotfix release

**Comment by Dyaxler** [comment]
Yep. Fixed. I finally got the SecureActionButton elements in the correct order. :D It's been a minute since I've used this template. Out of combat, Blizzard doesn't care which order the elements are applied. In Combat, very specific. Especially the SetAttribute() element.

https://github.com/Questie/Questie/commit/08b40feade5537cf4c1dcfb6eaeb593953f551a5

**Comment by Dyaxler** [comment]
@Gymson64 new version of Questie is out. Please retest. :)
